modules/300-prometheus/template_tests/secret_datasources_list_test.go

```diff
@@ -122,6 +110,7 @@ internal:
 		"Grafana datasources secret was rendered correctly",
 		func(haEnabled bool, longtermRetentionDays, countSources, countDeleted int) {
 			f.ValuesSetFromYaml("global", getGlobalValues(haEnabled))
+			f.ValuesSet("global.modulesImages", GetModulesImages())
```

Comment:
don't need here

Reply:
done

Reply:
reverted due to failed test
https://github.com/deckhouse/deckhouse-test-1/actions/runs/3154314187/jobs/5131887278